### PR TITLE
Trying to build against votca without building ctp

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
Removing support for ctp through building on gitlab runner.

### CODE REVIEW

**Comment by JoshuaSBrown** [comment]
@junghans if you have any advice on how to do this I would be indepted, this is driving me crazy. I don't understand why the lines that I changed are now causing problems with gitlabs ability to find the votca tools.

**Comment by JoshuaSBrown** [comment]
Do I need the CMakeModule for VOTCA TOOLS because I tried that...?

File: CMakeLists.txt [hunk 1]
```diff
@@ -82,7 +82,7 @@ if(NOT TARGET manpages)
   add_custom_target(manpages ALL)
 endif()
 
-find_package(VOTCA_TOOLS REQUIRED NO_MODULE)
+find_package(VOTCA_TOOLS NO_MODULE REQUIRED)
```
Comment (JensWehner) [inline CMakeLists.txt]: why that?

**Comment by JensWehner** [comment]
I really have not too much expertise in this.

File: CMakeLists.txt (hunk 1 above)
Comment (JoshuaSBrown) [reply CMakeLists.txt]: Umh... how about become I don't know what I'm doing with cmake and the documentation and examples are lacking even though its been around for years.

**Comment by JoshuaSBrown** [comment]
The problem is that I was trying to build against votca/electrostatics which does not exist so I think it was defaulting to master. I can get around this byusing for/development/

**Pull request closed without merging**